src/components/Login/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { Redirect } from "react-router-dom";
+
+export default class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isAuthenticated: false,
+      isShowingError: false,
+      userName: "",
+      password: "",
+    };
+
+    this.handleLogin = this.handleLogin.bind(this);
+  }
+
+  updateField(e, field) {
+    this.setState({
+      [field]: e.target.value,
+    });
+  }
+
+  handleLogin() {
+    const { userName, password } = this.state;
+
+    if (userName === "ziv" && password === "papa") {
+      this.setState(prevState => ({
+        isAuthenticated: !prevState.isAuthenticated,
+      }));
+    } else {
+      this.setState(prevState => ({
+        isShowingError: !prevState.isShowingError,
+      }));
+    }
+  }
+
+  render() {
+    const { isAuthenticated, isShowingError } = this.state;
+
+    return isAuthenticated ? (
+      <Redirect to="/bounce_rules" />
```

Comment:
store `"/bounce_rules"` in a variable in case this route needs to be changed in the future